Website/js/Debug/dnn.modalpopup.js

```diff
@@ -201,29 +201,41 @@
                     var $dnnToggleMax = $('<a href="#" class="dnnToggleMax"><span>Max</span></a>');
                     $modal.parent().find('.ui-dialog-titlebar-close').before($dnnToggleMax);
 
-                    $dnnToggleMax.click(function(e) {
+                    var initialTop = $modal.parent().css("top");
+                    var initialLeft = $modal.parent().css("left");
+                    
+                    $dnnToggleMax.click(function (e) {
                         e.preventDefault();
 
                         var $window = $(window),
                             newHeight,
-                            newWidth;
+                            newWidth,
+                            newTop,
+                            newLeft;
 
                         if ($modal.data('isMaximized')) {
-                            newHeight = $modal.data('height');
+                            newHeight = $modal.data('height') + 100;
                             newWidth = $modal.data('width');
                             $modal.data('isMaximized', false);
+
+                            newTop = initialTop;
+                            newLeft = initialLeft;
+
                         } else {
                             $modal.data('height', $modal.dialog("option", "minHeight"))
-                                .data('width', $modal.dialog("option", "minWidth"))
-                                .data('position', $modal.dialog("option", "position"));
+                                .data('width', $modal.dialog("option", "minWidth"));
 
                             newHeight = $window.height() - 46;
                             newWidth = $window.width() - 40;
+
+                            newTop = "2%";
+                            newLeft = "1%";
+
                             $modal.data('isMaximized', true);
                         }
 
                         $modal.dialog({ height: newHeight, width: newWidth });
-                        $modal.dialog({ position: { my: "center", at: "center" } });
+                        $modal.parent().css({ "top": newTop, "left": newLeft, "position": "fixed" });
```

Comment:
I have some concern about this fix. We should stick to the jQuery UI API as much as possible. I did a quick test and changing the calls in this way works to me in IE and Chrome:
    $modal.dialog( "option", "height", newHeight);
    $modal.dialog( "option", "width", newWidth);
    $modal.dialog( "option", "position", { my: "center", at: "center", of: window });
see http://api.jqueryui.com/dialog/#option-position

With this solution we do not need store the initial top and left + change the position to fixed. I do not see why we want have fixed here, we can talk on this today. But IMO we should always have dialog fixed or absolute and try to do not change it only when maximize or minimize.